guides/v2.1/cloud/cdn/fastly-vcl-badreferer.md

```diff
@@ -11,57 +11,104 @@ functional_areas:
-* Account access and URL to the Magento Admin for the Staging or Production environment
+**Prerequisites**
 
-{:.bs-callout .bs-callout-info}
-This information is just the code portion for setting up your VCL. Use this information with [Custom Fastly VCL snippets]({{ page.baseurl }}/cloud/cdn/cloud-vcl-custom-snippets.html).
+-  Configure your {{ site.var.data.ece }} environment for Fastly services. See [Set up Fastly]({{ page.baseurl }}/cloud/cdn/configure-fastly.html). 
 
-## Create an Edge Dictionary {#edge-dictionary}
+-  Get credentials to access the Admin UI for your {{ site.data.var.ece }} environment.
 
-Edge Dictionaries create key-value pairs for running against your VCL snippet. For example, you may want to build a dictionary of URLs to redirect to a Wordpress backend. You may only want to create the edge dictionary in your Production environment. You can also create it in Staging for testing if needed.
+-  Review your site log for traffic spamming your site and note the domain names that you need to block.
 
-1. Log in to the Magento Admin.
-2. Navigate to **Stores** > **Configuration** > **Advanced** > **System** > **Fastly Configuration**.
-3. Expand the **Edge dictionaries** section.
-4. Click **Add container**. You need to create a container to hold up to 1,000 key-value pairs.
-5. On the container, enter a Dictionary name. For this example, use the name `referer_blocklist`.
-6. Select the checkbox for **Activate after the change** if you want to the dictionary after creating or editing the container.
-7. Add key-value pairs in the new dictionary. For this example, enter the URLs for your blog that should be redirected to your Wordpress backend. Enter a value of 1.
+
+## Create a referrer block list
+
+Edge Dictionaries create key-value pairs accessible to VCL functions during VCL snippet processing. In this example, you create an edge dictionary that provides the list of website referrers that you want to block.
+
+1.  Log in to the Admin UI for your {{ site.data.var.ece }} project environment.
+
+1.  Navigate to **Stores** > **Configuration** > **Advanced** > **System** > **Fastly Configuration**.
+
+1.  Expand the **Edge dictionaries** section.
+
+1.  Create the Dictionary container:
+
+    - Click **Add container**.
+
+    -  On the *Container* page, enter a **Dictionary name**. For this example, use the name `referrer_blocklist`.
+
+    -  Select **Activate after the change** to deploy your changes to the service version that you are editing.
+
+    -  Click **Upload** to attach the dictionary to your Fastly service configuration.
+
+1.  Add the list of domain names to block to the the `referrer_blocklist` dictionary:
+
+    -  Click the Settings icon for the `referrer_blocklist` dictonary.
+
+    -  Add and save key-value pairs in the new dictionary. For this example, each **Key** is a website referrer URL that you want to block and the **Value** is `true`. 
+       
+       ![Add bad referrer dictionary items]
```

Comment:
Although this graphic is useful, I do find that the top of the graphic contains text that is the same size as the step text above...it's strange how it looks like another step. Is there a way to present this graphic better?